src/Kafka/CreateKafkaInstance/Stories/FormLoad.test.tsx

```diff
@@ -378,4 +410,235 @@ describe("CreateKafkaInstanceWithSizes", () => {
       ).toBeInTheDocument()
     );
   });
+
+  it("should show the billing options, with prepaid and a single subscription", async () => {
+    const comp = renderDialog(<PrepaidAndSingleMarketplaceSubscription />);
+    await waitForI18n(comp);
+
+    await waitFor(() =>
+      expect(
+        comp.queryByText("Checking if new Kafka instances are available")
+      ).not.toBeInTheDocument()
+    );
+
+    await waitFor(() => expect(comp.getByText("Billing")).toBeInTheDocument());
+    await waitFor(() =>
+      expect(comp.getByText("Red Hat pre-paid")).toBeInTheDocument()
+    );
+    await waitFor(() =>
+      expect(
+        comp.getByText("Amazon Web Services Marketplace")
+      ).toBeInTheDocument()
+    );
+
+    userEvent.selectOptions(
+      comp.getByLabelText("Cloud provider *"),
+      "Microsoft Azure"
+    );
+    await waitFor(async () =>
+      expect(
+        (
+          await comp.findByText("Amazon Web Services Marketplace")
+        ).closest("[role=option]")
+      ).toHaveAttribute("aria-disabled", "true")
+    );
+  });
+
+  it("should show the billing options, with prepaid and a single subscription and save the right data", async () => {
+    const onCreate = jest.fn();
+    const comp = renderDialog(
+      <PrepaidAndSingleMarketplaceSubscription onCreate={onCreate} />
+    );
+    await waitForI18n(comp);
+
+    await waitFor(() =>
+      expect(comp.queryByTestId("size-loading")).not.toBeInTheDocument()
+    );
+
+    userEvent.type(comp.getByLabelText("Name *"), "test-name");
+    userEvent.click(comp.getByText("Red Hat pre-paid"));
+    userEvent.click(comp.getByRole("button", { name: "Create instance" }));
+    await waitFor(() =>
+      expect(onCreate).toHaveBeenCalledWith(
+        expect.objectContaining({
+          billing: "prepaid",
+          name: "test-name",
+          provider: "aws",
+          region: "eu-west-1",
+          sizeId: "x1",
+        }),
+        expect.anything(),
+        expect.anything()
+      )
+    );
+  });
+
+  it("should show the billing options, with prepaid and multiple subscriptions", async () => {
+    const comp = renderDialog(<PrepaidAndMarketplaceSubscriptions />);
+    await waitForI18n(comp);
+
+    await waitFor(() =>
+      expect(
+        comp.queryByText("Checking if new Kafka instances are available")
+      ).not.toBeInTheDocument()
+    );
+
+    await waitFor(() => expect(comp.getByText("Billing")).toBeInTheDocument());
+    await waitFor(() =>
+      expect(comp.getByText("Red Hat pre-paid")).toBeInTheDocument()
+    );
+    await waitFor(() =>
+      expect(comp.getAllByText("Amazon Web Services Marketplace")).toHaveLength(
+        2
+      )
+    );
+    await waitFor(() => {
+      const azure = comp.getByText("Microsoft Azure Marketplace");
```

Comment:
Similar to changing "Amazon Web Services Marketplace" to just "AWS Marketplace", this should just be "Azure Marketplace" if we follow how MS presents this label: https://azuremarketplace.microsoft.com/en-us/marketplace/

Reply:
done